metrics/metric_template.go

```diff
@@ -46,12 +46,15 @@ func (mv *Vars) Compose(providedTemplate string) (string, error) {
 }
 
 func (mv *Vars) Parse(stream *Stream) {
+	space, _ := stream.App.Space()
+	org, _ := space.Org()
```

Comment:
These two methods each generate an API request. As I understand, `Stream()` is called for every metric, so this would be more expensive than the problem addressed by #5.

Can you see a way to pre-load these when we start monitoring an app?